modules/openshift-architecture-common-terms.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-architecture-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+access policies::
+A set of roles that dictate how users, applications, and entities within a cluster interacts with one another. Access policies increase cluster security.
```

Comment:
I think you should use an article before access policies here because access is a verb but its being used as a noun in this sentence. Maybe: An access policy increases cluster security.

Reply:
Addressed